src/index.ts

```diff
@@ -1,11 +1,49 @@
 import { Probot } from "probot";
 
 export = (app: Probot) => {
+  
+  // Respond to an issue opened 
   app.on("issues.opened", async (context) => {
     const issueComment = context.issue({
       body: "Thanks for opening this issue!",
     });
     await context.octokit.issues.createComment(issueComment);
+  });  
+  // Respond to the GitHub app installation 
+  app.on("installation.created", async (context) =>{
+
+    // TODO: get the installation token from the url
+    const installationTokenUrl = context.payload.installation.access_tokens_url;
+    const installationToken = await context.octokit.request("Get {url}",{url: installationTokenUrl});
+    app.log.info("installationt token: {}",installationToken);
+    // TODO: iterate over the list of repositories
+    const repos = context.payload.repositories;
```

Comment:
We want to check if there's a `.github` repository in the org, and if not, create it.